test/test_app_SUITE.erl

```diff
@@ -21,7 +21,8 @@ test_app(_Config) ->
             _ ->
                 {ignore, ["src/*_ignore.erl", "src/ignored_file_config.erl", "src/otp23.erl"]}
         end,
-    State2 = rebar_state:set(State1, format, [Files, IgnoredFiles]),
+    Options = {options, #{spaces_around_fields => true}},
```

Comment:
Why is this option needed here? I think that the default (`false`) is ok for tests.